cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/AbstractSparkContext.java

```diff
@@ -257,7 +262,11 @@ public DiscoveryServiceClient getDiscoveryServiceClient() {
    * not serializable and shouldn't be exposed to user program directly.
    */
   public Metrics getUserMetrics() {
```

Comment:
why is this called getUserMetrics() if the resulting metrics instance should not be exposed to users?